<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(32, 36, 38);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="headerFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10000</width>
        <height>40</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QFrame{
border-top-left-radius: 10px;
border-top-right-radius:10px;
border-bottom-left-radius: 0px;
border-bottom-right-radius:0px;
background-color: rgb(61, 64, 70);
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="minBtn">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>11</y>
         <width>34</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: rgb(56, 57, 61);
}
QPushButton{
	border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/min ver 2.png</normaloff>images/min ver 2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="closeBtn">
       <property name="geometry">
        <rect>
         <x>864</x>
         <y>11</y>
         <width>28</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: rgb(56, 57, 61);
}
QPushButton{
	border-radius:10px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/outline_close_white_48dp.png</normaloff>images/outline_close_white_48dp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="title_label">
       <property name="geometry">
        <rect>
         <x>347</x>
         <y>7</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft PhagsPa</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(214, 214, 214);</string>
       </property>
       <property name="text">
        <string>Math Visualisation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="window_icon">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>7</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: rgb(56, 57, 61);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/icon.png</normaloff>../images/icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 3px;
border-color:rgb(61, 64, 70);
background-color: rgb(32, 33, 36);
QFrame{
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="top_menu_frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom:0px solid;
border-radius:10px;

background-color: rgb(81, 85, 93);
border-top:0px solid;
border-left:0px solid;
border-right:0px solid;
border-color: rgb(91, 91, 91);
border-radius:0px;
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="math_text_frame">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>50</y>
            <width>871</width>
            <height>70</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="menu_buttons_frame">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>891</width>
            <height>43</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
border-bottom: 1px solid #dadada;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="math_symbol_button">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>0</y>
             <width>41</width>
             <height>41</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft PhagsPa</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover {
	background-color: rgb(56, 57, 61);
}
QToolTip{color: white;}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/calculate.png</normaloff>
             <normalon>../images/calculate.png</normalon>images/calculate.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="tooltip_label">
           <property name="geometry">
            <rect>
             <x>550</x>
             <y>0</y>
             <width>331</width>
             <height>41</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{

border-bottom: 0px;

color:rgb(214, 214, 214);

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="add_text_input_btn">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>0</y>
             <width>41</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{background-color:rgb(56, 57, 61);}
QToolTip{color: white;}
QPushButton{border-radius: 10px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../images/new.png</normaloff>../images/new.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>41</width>
             <height>41</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="delete_all_btn">
           <property name="geometry">
            <rect>
             <x>95</x>
             <y>0</y>
             <width>41</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{background-color:rgb(56, 57, 61);}
QToolTip{color: white;}
QPushButton{border-radius: 10px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../images/baseline_delete_white_36dp.png</normaloff>
             <normalon>../images/outline_close_white_48dp.png</normalon>../images/baseline_delete_white_36dp.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>41</width>
             <height>41</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="help_btn">
           <property name="geometry">
            <rect>
             <x>138</x>
             <y>0</y>
             <width>41</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../images/outline_help_white_48dp.png</normaloff>../images/outline_help_white_48dp.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QLineEdit" name="input_edit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>124</y>
            <width>871</width>
            <height>25</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>871</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>871</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{border:2px solid;

border-color:rgb(214, 214, 214);
background-color: rgb(81, 85, 93);
color:rgb(214, 214, 214);}</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="input_scroll">
         <property name="styleSheet">
          <string notr="true">border-width: 0px;</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>894</width>
            <height>795</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QStackedWidget" name="stackedWidget">
             <property name="styleSheet">
              <string notr="true">border-radius: 0px;</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="output_frame">
                 <property name="minimumSize">
                  <size>
                   <width>700</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame{border:2px solid;

border-color:rgb(214, 214, 214);
background-color: rgb(81, 85, 93);
color:rgb(214, 214, 214);}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_2"/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="actionSave_Images">
   <property name="text">
    <string>Save Plots..</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionblah_blah">
   <property name="text">
    <string>blah blah</string>
   </property>
  </action>
  <action name="actionstuff">
   <property name="text">
    <string>stuff</string>
   </property>
  </action>
  <action name="actionstuff_2">
   <property name="text">
    <string>stuff</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
